lib/Tags/TagsAccordion.js

```diff
@@ -32,6 +32,7 @@ const TagsAccordion = React.memo(({ link, getEntity, getEntityTags, entityTagsPa
             <FormattedNumber value={entityTags.length} />
           </Badge>
         )}
+        id={id}
```

Comment:
Have you tried to use `closedByDefault` prop? IMHO, it's much more clear solution

Reply:
Maybe it's better to add `{...rest}` props to the accordion to have the possibility to pass other props?

Reply:
I think it's a common requirement - to show accordion opened initially if some tags are selected. Maybe you can check `entityTags` for length